### ISSUE TITLE
Fix a11y workflow

### ISSUE THREAD

**Issue by allie500** [allie500/emoji-match-game#33]
# Description

Recently the `a11y` Github action is failing at the _Install Chrome for accessibility scans_ step. Below is the output from the latest run.

<details><summary><strong>A11y workflow install Chrome step log</strong></summary>

```bash
emoji-match-game@0.0.16 a11y:install-chrome
> npx puppeteer browsers install chrome

puppeteer browsers install [browser]

Download and install the specified browser. If successful, the command outputs the actual browser buildId that was installed and the absolute path to the browser executable (see --format).

Positionals:
  browser  Which browser to install <browser>[@<buildId|latest>]. `latest` will try to find the latest available build. `buildId` is a browser-specific identifier such as a version or a revision.  [string]

Options:
  --version       Show version number  [boolean]
  --help          Show help  [boolean]
  --platform      Platform that the binary needs to be compatible with.  [string] [choices: "linux", "linux_arm", "mac", "mac_arm", "win32", "win64"] [default: Auto-detected]
  --base-url      Base URL to download from  [string]
  --install-deps  Whether to attempt installing system dependencies (only supported on Linux, requires root privileges).  [boolean] [default: false]
  --format        Format to use for the output. Supported placeholders: {{browser}}, {{buildId}}, {{path}}, {{platform}}  [string] [default: "{{browser}}@{{buildId}} {{path}}"]

Examples:
  puppeteer install                             Install all pinned browsers
  puppeteer install chrome                      Install the pinned available build of the Chrome browser.
  puppeteer install chrome@latest               Install the latest available build for the Chrome browser.
  puppeteer install chrome@stable               Install the latest available build for the Chrome browser from the stable channel.
  puppeteer install chrome@beta                 Install the latest available build for the Chrome browser from the beta channel.
  puppeteer install chrome@dev                  Install the latest available build for the Chrome browser from the dev channel.
  puppeteer install chrome@canary               Install the latest available build for the Chrome Canary browser.
  puppeteer install chrome@115                  Install the latest available build for Chrome 115.
  puppeteer install chromedriver@canary         Install the latest available build for ChromeDriver Canary.
  puppeteer install chromedriver@115            Install the latest available build for ChromeDriver 115.
  puppeteer install chromedriver@115.0.5790     Install the latest available patch (115.0.5790.X) build for ChromeDriver.
  puppeteer install chrome-headless-shell       Install the latest available chrome-headless-shell build.
  puppeteer install chrome-headless-shell@beta  Install the latest available chrome-headless-shell build corresponding to the Beta channel.
  puppeteer install chrome-headless-shell@118   Install the latest available chrome-headless-shell 118 build.
  puppeteer install chromium@1083080            Install the revision 1083080 of the Chromium browser.
  puppeteer install firefox                     Install the latest nightly available build of the Firefox browser.
  puppeteer install firefox@stable              Install the latest stable build of the Firefox browser.
  puppeteer install firefox@beta                Install the latest beta build of the Firefox browser.
  puppeteer install firefox@devedition          Install the latest devedition build of the Firefox browser.
  puppeteer install firefox@esr                 Install the latest ESR build of the Firefox browser.
  puppeteer install firefox@nightly             Install the latest nightly build of the Firefox browser.
  puppeteer install firefox@stable_111.0.1      Install a specific version of the Firefox browser.
  puppeteer install firefox --platform mac      Install the latest Mac (Intel) build of the Firefox browser.

Error: All providers failed for chrome 147.0.7727.57:
  - DefaultProvider: The browser folder (/home/runner/.cache/puppeteer/chrome/linux-147.0.7727.57) exists but the executable (/home/runner/.cache/puppeteer/chrome/linux-147.0.7727.57/chrome-linux64/chrome) is missing
    at installWithProviders (/home/runner/work/emoji-match-game/emoji-match-game/node_modules/@puppeteer/browsers/lib/cjs/install.js:119:11)
    at async install (/home/runner/work/emoji-match-game/emoji-match-game/node_modules/@puppeteer/browsers/lib/cjs/install.js:129:12)
    at async #install (/home/runner/work/emoji-match-game/emoji-match-game/node_modules/@puppeteer/browsers/lib/cjs/CLI.js:351:9)
    at async Object.handler (/home/runner/work/emoji-match-game/emoji-match-game/node_modules/@puppeteer/browsers/lib/cjs/CLI.js:228:17)
Error: Process completed with exit code 1.
```

</details>

### Acceptance criteria 

- Update the workflow's command to install Chrome so that the Github action completes.

### Dev notes

[Failed a11y job run](https://github.com/allie500/emoji-match-game/actions/runs/27592420898/job/81932424444?pr=32)
